### DISCUSSION TITLE
CA certificate not found.

### DISCUSSION THREAD

**Discussion by gour** [GothenburgBitFactory/taskwarrior#2722] [Community support] [answered]
Hello,

I'm back (from org-mode to Taskwarrior), re-configured everything from the scratch on my Debian/Sid machine, but experience weird behavior that after initial sync, `task sync` sometimes does work and then, without  any change from my side, it reports about missing certs:

```$  task diag

task 2.6.1
   Platform: Linux

Compiler
    Version: 11.2.0
       Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64
 Compliance: C++17

Build Features
      CMake: 3.21.3
    libuuid: libuuid + uuid_unparse_lower
  libgnutls: 3.7.2
 Build type: None

Configuration
       File: /home/gour/.config/task/taskrc (found), 2003 bytes, mode 100644
       Data: /home/gour/task (found), dir, mode 40755
    Locking: Enabled
         GC: Enabled
    $EDITOR: vim
     Server: atmarama.ddns.net:53589
         CA: not found
Certificate: not found
        Key: not found
      Trust: strict
    Ciphers: NORMAL
      Creds: Atmarama/gour/************************************

Hooks
     System: Enabled
   Location: /home/gour/task/hooks
             (-none-)

Tests
   Terminal: 200x62
       Dups: Scanned 9886 tasks for duplicate UUIDs:
             No duplicates found
 Broken ref: Scanned 9886 tasks for broken references:
             No broken references found
```

Here is the snippet from my config file:

```
...
taskd.certificate=gour.cert.pem
taskd.key=gour.key.pem
taskd.ca=ca.cert.pem
...
```

and certs are there:

```
$ ll task/*.pem
-rw------- 1 gour gour 2,0K sij  20 15:28 task/ca.cert.pem
-rw------- 1 gour gour 2,0K sij  20 15:39 task/gour.cert.pem
-rw------- 1 gour gour  11K sij  20 15:39 task/gour.key.pem
```

Any hint or shall I open a new ticket?

**Comment by gour** [comment, marked as answer]
It looks like the problem in 2.6.x is the following:

```
...
taskd.certificate=gour.cert.pem
taskd.key=gour.key.pem
taskd.ca=ca.cert.pem
...
```

iow. after adding full path like `taskd.certificate=~/task/gour.cert.pem` everything is fine. :smile: 

However, I haven't found any info in release notes in regard, while the old setup was working in 2.5.x. :confused:

**Reply by djmitche** [reply]
I wonder, was the old setup working because you were running from the `task` directory?

**Reply by gour** [reply]
> I wonder, was the old setup working because you were running from the `task` directory?

Well, my setup is indeed in `~/task`, but it is just a symlink and I **do not** run from the `~/task`, but using `tasksh` from `~/`?
